tools/build-manylinux.sh

```diff
@@ -73,13 +73,10 @@ done
 
 # Install packages and test
 echo "# Installing wheels"
-for PYBIN in /opt/python/*/bin/; do
+for PYBIN in /opt/python/cp*/bin/; do
     echo "## Installing $PYBIN"
     "${PYBIN}/pip" install confluent_kafka -f /io/wheelhouse
     "${PYBIN}/python" -c 'import confluent_kafka; print(confluent_kafka.libversion())'
     echo "## Uninstalling $PYBIN"
     "${PYBIN}/pip" uninstall -y confluent_kafka
 done
```

Comment:
Add a new line at the end of the files to adhere to the styling followed in the repo.

Reply:
Check other files in the PR as well

Reply:
Updated.